zone_source/iw4x_code_post_gfx_mp.csv

```diff
@@ -0,0 +1,73 @@
+// Call Of Duty: Modern Warfare 2
+
+structureddatadef,mp/playerdata.def
```

Comment:
My code post gfx MP has more things than this, this seems to be missing a lot of menus. Aswell as a TXT menufile. Are you sure this is OK?

Reply:
```
.\bin\Unlinker.exe .\iw4x_code_post_gfx_mp.ff
```

Reveals a zone with one asset only (the structured data definition). The fast file was modified by OAT the last time Laupetin touched it. Safe to say it's all good. I think your source files are wrong.

![image](https://github.com/iw4x/iw4x-rawfiles/assets/37080671/26bab9f8-f2cb-42e6-bfcb-64c34a606f6d)